src/PPN-microbench/_ops/ops.cpp

```diff
@@ -0,0 +1,117 @@
+#include <PPN-microbench/_ops/ops.hpp>
+
+using std::chrono::duration_cast;
+using std::chrono::nanoseconds;
+using std::chrono::steady_clock;
+
+Ops::Ops(int reps) : Microbench("OPS", reps) {
+    cpus = context.getJson()["cpu_info"]["cpus"];
+    results = new size_t *[6];
```

Comment:
Why use `new`? A vector of static arrays is fine as the size of the elements are known at compile-time (it would be like having a vector of 6-fields `struct`s).